### ISSUE TITLE
Vite bundling error

### ISSUE THREAD

**Issue by DuncanLHS** [JaleelB/emblor#68]
I'm having trouble using emblor in a Remix (vite) project. Getting the following errors:

In the vite dev server:
```
[vite] Error when evaluating SSR module {filepath}/form.tsx: failed to import "emblor"
```

In the browser:
```
Named export 'SortableItem' not found. The requested module 'react-easy-sort' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'react-easy-sort';
```

I believe vite is supposed to automatically convert commonJS modules but it's not playing nice in this case. Any pointers?

**Comment by ersinakinci** [comment]
I'm having this problem, too.

**Comment by DuncanLHS** [comment]
OK, I think I've solved it. I thought I'd tried this already and it didn't work as I had to do the same with tailwind-merge but adding the right packages to noExternal in my Vite config seems to have done the trick.

```ts
ssr: {
    noExternal: ["tailwind-merge", "emblor", "react-easy-sort", "tslib"],
  },
  resolve: {
    alias: {
      tslib: "tslib/tslib.es6.js",
    },
  },
```

Just be sure to delete the Vite cache in `node_modules/.vite` before restarting your dev server. This has also deployed to vercel correctly.
